Cue/app/tabs/library/play/PlayDeckView.js

```diff
@@ -145,6 +153,24 @@ class PlayDeckView extends React.Component {
     StatusBar.setHidden(false)
   }
 
+  _flaggedOnly(array: Array<*>): Array<*> {
```

Comment:
I would actually specify the generic parameter here and in `_answerFirst` as `Deck` since we're actually requiring that this is an array of `Deck` objects, rather than just any old array of objects.